tests/integration_test/services_test.py

```
@@ -1456,6 +1456,191 @@ def test_list_bots():
+    # Verify update — empid should remain unchanged (1234), name should change
+    list_resp_after_update = client.post(
+        f"/api/bot/{bot_id}/collections/data",
+        headers={"Authorization": pytest.token_type + " " + pytest.access_token},
+        json={"collection_name": "testing_create_colection_secure"}
+    )
+    updated_doc = next(doc for doc in list_resp_after_update.json()["data"] if doc["id"] == doc_id)
+    assert updated_doc["data"]["name"] == "testing_updated"
+    assert updated_doc["data"]["empid"] == 1234  # Unchanged due to is_non_editable
+
+    # Step 5: Delete the document
+    delete_payload = {
```

Ruff (0.11.9)

Local variable `delete_payload` is assigned to but never used

Remove assignment to unused variable `delete_payload`

(F841)